crates/compiler/gen_llvm/src/llvm/bitcode.rs

```diff
@@ -30,12 +30,22 @@ pub fn call_bitcode_fn<'ctx>(
     args: &[BasicValueEnum<'ctx>],
     fn_name: &str,
 ) -> BasicValueEnum<'ctx> {
-    call_bitcode_fn_help(env, args, fn_name)
+    let ret = call_bitcode_fn_help(env, args, fn_name)
         .try_as_basic_value()
         .left()
         .unwrap_or_else(|| {
             panic!("LLVM error: Did not get return value from bitcode function {fn_name:?}")
-        })
+        });
+
+    if env.target_info.operating_system == roc_target::OperatingSystem::Windows {
+        // On windows zig uses a vector type <2xi64> instead of a i128 value 
```

Comment:
I thought that is what old zig did and it changed to proper i128 with 0.11.0. I guess not though.

Reply:
Yeah, I'm not sure why. Maybe it will change in future with a later version of zig.

Reply:
a later LLVM version changes the alignment of i128 to 16. My guess is that guaranteeing an alignment of 8 is what is at play here